### ISSUE TITLE
Some Portuguese translation strings require a double check

### ISSUE THREAD

**Issue by fergie-nz** [msupply-foundation/open-msupply#4974]
## Describe the solution you'd like 🎁

We are needing checking of the following strings where there is some ambiguity:

key | option 1 | option 2
-- | -- | --
label.supply-quantity | Fornecer | Quantidade a fornecer
label.requested-quantity | Solicitado | Quantidade solicitada
label.counted-num-of-packs | Pacotes Contados | Número de pacotes contados
messages.select-rows-to-delete | Selecione as linhas para excluí-las | Selecione as linhas para serem excluídas
messages.no-related-documents | Não existem documentos relacionados | Nenhum documento relacionado

There are some other strings which have been differentiated or consolidated with a bit more confidence. It would still be helpful if you're willing to give these a look to make sure they're correct:

Differentiated:

key 1 | translation | new alternate key | alternate translation
-- | -- | -- | --
error.no-items | Nenhum item | error.no-items-to-display | Não existem itens para mostrar. 
label.months | meses | text.months | Meses
label.stock-on-hand-remaining | Estoque Disponível (Restante estimado) | label.stock-on-hand | Estoque Disponível
error.requisition-not-found | Requisição não foi encontrada | error.order-not-found | Requisição Interna não encontrada
label.amc | CMM | label.amc/amd | CMM/AMD

And consolidated into one key (removing one of the translations:

key | option one | option two | consolidated to:
-- | -- | -- | --
filename.asset-categories | categorias-bens | Categoria de bens | categorias-bens

**Comment by andreievg** [comment]
@yeisonportela can you please have a look at these in weblate

**Comment by yeisonportela** [comment]
> @yeisonportela can you please have a look at these in weblate

Yes Andrei of course, will see this with Tainá

**Comment by regotaina** [comment]
Hi team

For the table below, Option 2 is more suitable for all alternatives.

key | option 1 | option 2
-- | -- | --
label.supply-quantity | Fornecer | Quantidade a fornecer
label.requested-quantity | Solicitado | Quantidade solicitada
label.counted-num-of-packs | Pacotes Contados | Número de pacotes contados
messages.select-rows-to-delete | Selecione as linhas para excluí-las | Selecione as linhas para serem excluídas
messages.no-related-documents | Não existem documentos relacionados | Nenhum documento relacionado

**Comment by regotaina** [comment]
For the following table, we can keep both alternatives. 'Display' = 'Mostrar' , and I can see the strings being used for different purposes. 

Unless you want to consolidate the 'error.no-items' with 'error.no-items-to-display'.
Is that the case?


key 1 | translation | new alternate key | alternate translation
-- | -- | -- | --
error.no-items | Nenhum item | error.no-items-to-display | Não existem itens para mostrar.

**Comment by regotaina** [comment]
In here we have Months capitalised, we can keep 'meses' for both with no changes to translation.
-- | -- | -- | --
label.months | meses | text.months | Meses

**Comment by regotaina** [comment]
Happy to keep both as it is:

key 1 | translation | new alternate key | alternate translation
-- | -- | -- | --
label.stock-on-hand-remaining | Estoque Disponível (Restante estimado) | label.stock-on-hand | Estoque Disponível
label.amc | CMM | label.amc/amd | CMM/AMD

**Comment by mark-prins** [comment]
it's the Internal Order detail view - when the internal order can't be found for the given ID

**Comment by regotaina** [comment]
> it's the Internal Order detail view - when the internal order can't be found for the given ID

after consulting the translate server, 'error.order-not-found' would be better translated to 'pedido interno não encontrado'

to be consistent with other strings already translated :) thanks, Mark!

**Comment by mark-prins** [comment]
updates made in weblate

**Comment by marthakeezy** [comment]
hey @regotaina does this need testing on your end or can it be moved to Done for QA? :)

**Comment by regotaina** [comment]
> hey [@regotaina](https://github.com/regotaina) does this need testing on your end or can it be moved to Done for QA? :)
 hey hey!

Can we see if the strings will be displayed correctly/ text wrapped accordingly? Or is this something we capture in the release cycle? happy to close this one if we already do some extra checks on how translations look like in the app, to ensure the words are not cut :)

**Comment by marthakeezy** [comment]
yep usually do that in the release cycle, i can move to Done if no context checking is needed - e.g. if all translations are correct!
